updating primary-remote on multi-net docs (#13917)
istio/istio.io: content/en/docs/setup/install/multicluster/primary-remote_multi-network/index.md

istio-eastwestgateway   LoadBalancer   10.0.12.121   34.122.91.98   ...       51s
{{< /text >}}

## Expose services in `cluster1` and `cluster2`

Since the clusters are on separate networks, we also need to expose all user

> services (*.local) on the east-west gateway in both clusters. While this
> gateway is public on the Internet, services behind it can only be accessed by

Reviewer: ```suggestion
services (*.local) on the east-west gateway in both clusters. While these
gateways are public on the Internet, services behind them can only be accessed by
```

services with a trusted mTLS certificate and workload ID, just as if they were
on the same network.

{{< text bash >}}
$ kubectl --context="${CTX_CLUSTER1}" apply -n istio-system -f \
    @samples/multicluster/expose-services.yaml@
{{< /text >}}

{{< tip >}}

> Since `cluster2` is installed with a remote profile. Exposing services on primary cluster will expose it on east-west gateways of both clusters.

Reviewer: ```suggestion
Since `cluster2` is installed with a remote profile, exposing services on the primary cluster will expose them on the east-west gateways of both clusters.
```
